docs/overview/metrics.md

```diff
@@ -2,63 +2,41 @@
 id: overview_metrics
 title: "Metrics"
 ---
-```scala mdoc:invisible
-import zio._
-import zio.random._
-import zio.duration._
-import zio.clock._
+ZIO ZMX enables the instrumentation of any ZIO based application with specialized aspects. The type of the original ZIO effect will not change by adding on or more aspects to it. 
 
-import zio.zmx.metrics._
-```
-ZMX allows the instrumentation of ZIO based applications with some extensions to the well known ZIO DSL. Using the DSL generates metrics events which will be processed 
-by a reporting backend that is registered as a layer within the application. Currently, reporting to Statsd and Prometheus is supported. It is important to note that 
-switching from one reporting backend to another does not require any code changes to the instrumented app. 
+Whenever an instrumented effect executes, all of the aspects will be executed as well and each of the 
+aspects will capture some data of interest und update some ZMX internal state. Which data will be captured and how it can be used later on is dependant on the metric type associated with the aspect. 
```

Comment:
```suggestion
aspects will capture some data of interest and update some ZMX internal state. Which data will be captured and how it can be used later on is dependant on the metric type associated with the aspect. 
```